packages/suite/src/reducers/suite/deviceReducer.ts

```diff
@@ -379,18 +379,9 @@ const addButtonRequest = (
 
 const setMetadata = (draft: State, state: string, metadata: TrezorDevice['metadata']) => {
     const index = draft.findIndex(d => d.state === state);
-    if (!draft[index]) return;
-    // update state
-    draft[index].metadata = metadata;
-};
-
-const updateMetadata = (draft: State, state: string, walletLabel?: string) => {
-    const index = draft.findIndex(d => d.state === state);
-    if (!draft[index]) return;
-    const { metadata } = draft[index];
-    if (metadata.status !== 'enabled') return;
-    // update state
-    metadata.walletLabel = walletLabel;
+    const device = draft[index];
```

Comment:
because I am not trying to make changes that are only relevant to what I am implementing here. I believe it helps us to focus on what really matters in the review